src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="big"
+    >
+      <input
+          type="search"
+          class="big_search input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+      >
```

Comment:
```suggestion
      <input
        type="search"
        class="big_search input"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```

Reply:
You have `big_search input` classes in one input and `small_search input` classes in the second one. What you need to do is to have one class `input` in the first input and two classes `small_search input` in the second one. You can find it in the checklist p.2.